internal/bloblang/query/functions.go

```diff
@@ -631,20 +632,28 @@ var _ = registerFunction(
 var _ = registerFunction(
 	NewFunctionSpec(
 		FunctionCategoryGeneral, "random_int",
-		"Generates a non-negative pseudo-random 64-bit integer. An optional integer argument can be provided in order to seed the random number generator.",
+		"Generates a non-negative pseudo-random 64-bit integer. An optional integer argument can be provided in order to seed the random number generator. Optional `min` and `max` arguments can be provided to make the generated numbers within a range.",
 		NewExampleSpec("",
 			`root.first = random_int()
-root.second = random_int(1)`,
+root.second = random_int(1)
+root.third = random_int(max:20)
+root.forth = random_int(min:10, max:20)
+root.fifth = random_int(timestamp_unix_nano(), 5, 20)
+root.sixth = random_int(seed:timestamp_unix_nano(), max:20)
+`,
 		),
 		NewExampleSpec("It is possible to specify a dynamic seed argument, in which case the argument will only be resolved once during the lifetime of the mapping.",
 			`root.first = random_int(timestamp_unix_nano())`,
+			`root.second = random_int(timestamp_unix_nano(), 5, 20)`,
 		),
 	).
 		Param(ParamQuery(
 			"seed",
 			"A seed to use, if a query is provided it will only be resolved once during the lifetime of the mapping.",
 			true,
-		).Default(NewLiteralFunction("", 0))),
+		).Default(NewLiteralFunction("", 0))).
+		Param(ParamInt64("min", "The minimum value the random generated number will have. The default value is 0").Default(0)).
+		Param(ParamInt64("max", "The maximum value the random generated number will have. The default value is 9223372036854775807").Default(int64(math.MaxInt64))),
```

Comment:
Nit: Instead of hardcoding that number, consider using `+ fmt.Sprint(math.MaxInt64)` instead

Reply:
Fixed it